arangod/Aql/ExecutionBlock.cpp

```diff
@@ -153,8 +158,9 @@ std::pair<ExecutionState, SharedAqlItemBlockPtr> ExecutionBlock::traceGetSomeEnd
   if (_profile >= PROFILE_LEVEL_BLOCKS) {
     ExecutionNode const* en = getPlanNode();
     ExecutionStats::Node stats;
+    auto const items = result != nullptr ? result->size() : 0;
     stats.calls = 1;
-    stats.items = result != nullptr ? result->size() : 0;
+    stats.items = items;
```

Comment:
why this?

Reply:
I now also include it in the logging a few lines further down: https://github.com/arangodb/arangodb/pull/10091/files/0bcdb26c26c160c6692de0f51446bbfd06a4038c#diff-f3f686cd0bf1f9e42ae0b2f834c2dbb4R183